cdap-ui/server/express.js

```diff
@@ -153,10 +153,15 @@ function makeApp (authAddress, cdapConfig, uiSettings) {
 
   app.get('/downloadLogs', function(req, res) {
     var url = decodeURIComponent(req.query.backendUrl);
+    var dataType = req.query.dataType;
+    var method = 'GET';
+    if (dataType === 'file') {
```

Comment:
instead of calling this `dataType`, should we just call the query parameter `method`, and let it default to `GET`, but will just use the method specified if it's provided?